egi_notebooks_hub/d4science.py

```diff
@@ -37,6 +38,13 @@
 D4SCIENCE_OIDC_URL = os.environ.get(
     "D4SCIENCE_OIDC_URL", "https://accounts.d4science.org/auth/realms/d4science/"
 )
+D4SCIENCE_INFOSYS_URL = os.environ.get(
```

Comment:
We have already a D4SCIENCE_DM_REGISTRY_URL which is basically pointing to another path of the icproxy..maybe you can just make configurable the base path

Reply:
done